functionaltests/src/test/java/specs/multitenancy/paypal/Authorization2Fixture.java

```diff
@@ -15,16 +15,18 @@ public MultiValueResult createPayment(
             final String paymentMethod,
             final String transactionType,
             final String centAmount,
-            final String currencyCode) throws Exception {
+            final String currencyCode) {
 
-        Payment payment = createAndSavePaypalPayment(paymentName, paymentMethod, transactionType, centAmount, currencyCode);
+        Payment payment = createAndSaveWalletPayment(paymentName, paymentMethod, transactionType,
+                centAmount, currencyCode,
+                "https://example.com/migration/paypal_authorization/");
```

Comment:
the provided URL is not tested, but is nice to have to debug tests with different payments